cluster/addons/rbac/cluster-loadbalancing/glbc/clusterrole.yaml

```diff
@@ -0,0 +1,24 @@
+# This was adapted from the canonical ClusterRole for GLBC can be found at:
+# https://github.com/kubernetes/ingress-gce/blob/master/docs/deploy/resources/rbac.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:controller:glbc
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
+- apiGroups: [""]
+  resources: ["secrets", "endpoints", "services", "pods", "nodes", "namespaces", "configmaps", "events"]
+  verbs: ["get", "list", "watch", "update", "create", "patch"]
+- apiGroups: ["extensions", "networking.k8s.io"]
+  resources: ["ingresses"]
+  verbs: ["get", "list", "watch", "update"]
+- apiGroups: ["extensions", "networking.k8s.io"]
+  resources: ["ingresses/status"]
+  verbs: ["update"]
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["get", "list", "watch", "update", "create", "patch"]
+- apiGroups: ["cloud.google.com"]
+  resources: ["backendconfigs"]
+  verbs: ["get", "list", "watch", "update", "create", "patch"]
```

Comment:
if this creates, I'd expect it to also delete. is that not the case?

Reply:
Not sure why we have create permission here although I hesitate to add / remove permissions here until we can do a proper audit and make sure we are not causing a regression. We can take on a TODO if that works.

Reply:
Added an issue and TODO to account/prune these permissions: https://github.com/kubernetes/ingress-gce/issues/743